tests/base/timer_03.cc

```diff
@@ -79,30 +79,36 @@ int main ()
   TimerOutput tO(std::cout,
                  TimerOutput::summary,
                  TimerOutput::cpu_times);
+
   tO.enter_section("Section1");
-  tO.enter_section("Section3");
-  tO.enter_section("Section4");
-  burn (50);
-  tO.exit_section("Section1");
-  tO.exit_section("Section4");
   tO.enter_section("Section2");
   burn (50);
   tO.exit_section("Section2");
-  tO.exit_section("Section3");
-  tO.enter_section("Section4");
+  tO.enter_section("Section2");
   burn (50);
-  tO.exit_section("Section4");
+  tO.exit_section("Section2");
+  tO.exit_section("Section1");
 
-  std::map<std::string, double> cpu_times = tO.get_summary_data(TimerOutput::OutputData::total_cpu_time);
-  std::map<std::string, double> calls = tO.get_summary_data(TimerOutput::OutputData::n_calls);
+  std::map<std::string, double> cpu_times =
+      tO.get_summary_data(TimerOutput::OutputData::total_cpu_time);
+  std::map<std::string, double> wall_times =
+      tO.get_summary_data(TimerOutput::OutputData::total_wall_time);
+  std::map<std::string, double> calls =
+      tO.get_summary_data(TimerOutput::OutputData::n_calls);
```

Comment:
@class4kayaker Always a good idea to check for the full interface you implemented. Thus, I added the third variant with `OutputData::total_wall_time` as well. :-)